workers/unity/Assets/Playground/Config/EntityTemplateUtils.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+
+namespace Playground
+{
+    public static class EntityTemplateUtils
+    {
+        public static readonly List<string> AllWorkerAttributes =
```

Comment:
This would be better as part of worker utils. However it's also worth noting that this is the worker types and is only the attribute set by coincidence. The worker names and attributes don't have to be the same.

Reply:
This seems to be something we conflate in a lot of places though.

Reply:
will move it into workerutils and rename to workertypes, if the user of the variable wants to use the worker types as attributes, it's on them (well us, but in a different context so I think the language works that way)

Reply:
In this case it makes sense to call it AllWorkerAttributes as that is how it's being used. But we should consider separating the type and layer into differnt variables.

Reply:
Ah I see

Reply:
I find it quite convenient in one way to not have to represent a worker with different kinds of identifiers. Would it make sense to have a different number of attributes than worker types? I am imagining having different kinds of `UnityClient`s or even `UnityGameLogic`s with different purposes (e.g. mobile can see one component and standalone can see another / fire UnityGameLogic can write to one component / ice UnityGameLogic can write to another component)? Should we create a ticket to distinguish better?